packages/konsole-app/src/config.ts

```diff
@@ -6,7 +6,7 @@ import * as config from '@dxos/config';
 
 // Generated by webpack-version-file-plugin.
 // eslint-disable-next-line import/no-unresolved
-import buildInfo from '../version.json';
+import buildInfo from '../dist/version.json';
```

Comment:
I don't think you should depend on build artifacts.

Reply:
But version.json is generated by webpack either way, this just changes the location and prevents version.json from being committed with each change.

PS: Sorry, this change should not be in this PR, I will rebase on #106